Grade.js

```diff
@@ -0,0 +1,19 @@
+let score = prompt("Enter The score");
+if(score>100){
+    console.log("Your score cannot be greater than 100");
```

Comment:
You really dont need this. This is for a school grade scale so no one would score over 100